app/components/Sidebar/index.tsx

```diff
@@ -9,22 +9,33 @@ import Link from 'next/link';
 import { usePathname } from 'next/navigation';
 import style from './sidebar.module.css';
 import { ROUTES } from '../../utils/routes';
+import {getCategories} from '../../services/dataMock.service';
+import {Category} from '../../interfaces/categories';
+import { useEffect } from 'react';
+
 export default function Sidebar() {
 
   const pathName = usePathname()
 
-  const [categories, setCategories] = useState(['Desayuno', 'Pastas', 'Guarniciones', 'Entradas'])
+  const [categories, setCategories] = useState<Category[]>()
```

Comment:
Very good! great that you used interfaces

Reply:
thank you!! i know you are proud of me!